config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
+  resources :games
```

Comment:
I think something is going wrong here, that's why there's a conflict. Did you delete 'resources :reservations' from the original file?

Reply:
naw I think me and Shouko were working on the two controllers at the same time. Once she finished the file that I was working on wasn't the original file anymore.
Should i jus add resources :reservations to my branch?

Reply:
I'll pull request it again

Reply:
That could be the problem, I'll look out for the new request

Reply:
ok i think we are good to go!!